openforcefield/tests/test_molecule.py

```diff
@@ -2764,6 +2764,43 @@ def test_get_bond_between(self):
         with pytest.raises(NotBondedError):
             mol.get_bond_between(hydrogens[0], hydrogens[1])
 
+    @pytest.mark.parametrize(
+        ("smiles", "n_rings"),
+        [
+            ("CCO", 0),
+            ("c1ccccc1", 1),
+            ("C1CC2CCC1C2", 2),  # This should probably be 3?
+            ("c1ccc(cc1)c2ccccc2", 2),
+            ("c1ccc2ccccc2c1", 2),
+            ("c1ccc2cc3ccccc3cc2c1", 3),
+            ("C1C2C(CCC1)CC5C3C2CCC7C3C4C(CCC6C4C5CCC6)CC7", 7),
+        ],
+    )
+    @requires_rdkit
+    def test_molecule_rings(self, smiles, n_rings):
+        """Test the Molecule.rings property"""
+        assert (
+            n_rings == Molecule.from_smiles(smiles, allow_undefined_stereo=True).n_rings
+        )
+
+    @requires_rdkit
+    def test_is_in_ring(self):
```

Comment:
This should probably be done using parametrize as well so we can easily work out which one is failing.